components/overflow-group/README.md

```diff
@@ -108,3 +124,9 @@ getOverflowContainer(overflowItems, mini) {
   else return html`<d2l-dropdown-button text="Overflow Menu"><d2l-dropdown-menu>${overflowItems}</d2l-dropdown-menu></d2l-dropdown-button>`;
 }
 ```
+
+## Accessibility
+
+Navigation and focus order are impacted when items overflow the viewport; they are physically moved into the More Actions menu. Since this effect is not limited to visual presentation, it affects both screen reader users and sighted users equally.
+
+The accessibility and keyboard interaction of the More Actions menu aligns with [W3C's best practices for menus](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/) — see [Menu](../../components/menu) for more information.
```

Comment:
To better match the changes we're making in the other component's documentation, should these be bullet points instead?

```suggestion
 * Navigation and focus order are impacted when items overflow the viewport; they are physically moved into the More Actions menu
  * Since this effect is not limited to visual presentation, it affects both screen reader users and sighted users equally
* The accessibility and keyboard interaction of the More Actions menu aligns with [W3C's best practices for menus](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/)
  *  See [Menu](../../components/menu) for more information on that component
```

Reply:
Also you should be able to shorten the link used for Menu to `../menu`

Reply:
Just closing the loop after talking over Slack: we decided the Accessibility sections should not have a particular format, and the format (bullets or paragraphs) should just be chosen to suit the content. There are already examples where we didn't use a bulleted list (like Calendar).